aws/data_source_aws_workspaces_image_test.go

```diff
@@ -0,0 +1,50 @@
+package aws
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceAwsWorkspacesImage_basic(t *testing.T) {
+	dataSourceName := "data.aws_workspaces_image.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccWorkspacesImagePreCheck(t)
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAwsWorkspacesImageConfig(os.Getenv("AWS_WORKSPACES_IMAGE_ID")),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(dataSourceName, "image_id", "wsi-vwtykwqyx"),
+					resource.TestCheckResourceAttr(dataSourceName, "name", "Windows10Baseline"),
+					resource.TestCheckResourceAttr(dataSourceName, "description", "Google Chrome AWS WorkDocs Drive CrowdStrike Falcon Russian/Ukrainian Language Pack"),
+					resource.TestCheckResourceAttr(dataSourceName, "os", "WINDOWS"),
+					resource.TestCheckResourceAttr(dataSourceName, "required_tenancy", "DEFAULT"),
+					resource.TestCheckResourceAttr(dataSourceName, "state", "AVAILABLE"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceAwsWorkspacesImageConfig(imageID string) string {
+	return fmt.Sprintf(`
+# TODO: Create aws_workspaces_image resource when API will be provided
+
+data "aws_workspaces_image" "test" {
+  image_id = %q
+}
+`, imageID)
+}
+
+func testAccWorkspacesImagePreCheck(t *testing.T) {
+	if os.Getenv("AWS_WORKSPACES_IMAGE_ID") == "" {
+		t.Fatal("AWS_WORKSPACES_IMAGE_ID env var must be set for AWS WorkSpaces image acceptance tests. This is required until AWS provides ubiquitous (Windows, Linux) import image API.")
```

Comment:
If the environment variable isn't set, this should skip the test instead of failing

Reply:
Fixed